modules/data.atmosphere/R/met_functions.R

```diff
@@ -24,6 +24,10 @@ get.latlonbox <- function(lati, loni, Lat = Lat, Lon = Lon) {
 
 get.cruncep <- function(lat, lon, start.date, end.date) {
   result <- load.cfmet(lat, lon)
+  Lat <- ncdf4::ncvar_get(nc = met.nc, varid = "lat")
+  Lon <- ncdf4::ncvar_get(nc = met.nc, varid = "lon")
+  lati <- which.min(abs(Lat - lat))
```

Comment:
I don't think think is right -- note that met.nc isn't an argument to this function, so I don't think it's available to read from here. 

But I also think something was broken here already -- `load.cfmet` has five mandatory arguments but is being passed only two of them on line 26.

Reply:
`get.weather`, `is.land`, `get.latlonbox`, and `get.cruncep` all are not exported and the latter three aren't called anywhere, while `get.weather` is called only in the very out-of-date `vignettes/compare_narr_cruncep_met.R`. I vote we delete all four of them.

Reply:
Deleted in 0b0d2434e